<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginItem</class>
 <widget class="QWidget" name="PluginItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	border:none;
	color:rgb(250,250,250);
	background:transparent;
}
#deleteButton:hover{
	border:none;
	border-radius:5px;
	background-color:rgb(250,250,250);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,12,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QToolButton" name="logoButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout" rowstretch="3,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QLabel" name="authorLabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>作者：xxx</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="versionLabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>版本：x.x</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="descriptionLabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>插件描述</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="nameLabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>插件名</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="typeLabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>#类别</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="sizeLabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>x.xxMB</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="deleteButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/resource/delete.png</normaloff>:/resource/delete.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
